configure.ac

```diff
@@ -19,7 +19,7 @@ AC_CONFIG_SRCDIR(src)
 AC_CONFIG_AUX_DIR(config)
 AC_CONFIG_MACRO_DIR(config)
 
-AM_INIT_AUTOMAKE(foreign [subdir-objects])
+AM_INIT_AUTOMAKE(foreign [subdir-objects no-dependencies])
```

Comment:
Please indicate if you have actually encountered VMA build failures due to dependencies usage